lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3468,6 +3468,24 @@ namespace {
 
       result->setHasUnreferenceableStorage(hasUnreferenceableStorage);
 
+      if (auto cxxRecordDecl = dyn_cast<clang::CXXRecordDecl>(decl)) {
+        result->setIsCxxNotTriviallyCopyable(
+            !cxxRecordDecl->isTriviallyCopyable());
+
+        for (const auto *ctor : cxxRecordDecl->ctors()) {
+          if (ctor->isCopyConstructor() && ctor->isDeleted()) {
+            result->setIsCxxNotTriviallyCopyable(true);
```

Comment:
Do you think we should also check that the copy constructor and destructor are public?

Reply:
Done